torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -941,30 +941,29 @@ c10::SymbolicShape ComputeShapeForSlice(
 }
 
 void ProcessSliceNode(Node* n, int opset_version) {
-  if (ConstantValueMap::HasShape(n->input(0)->debugName())) {
+  auto valid = true;
+  if (opset_version >= 10) {
```

Comment:
please add a comment stating why this if is here.

Reply:
added comments.